packages/dx-react-grid/docs/guides/filtering.md

```diff
@@ -11,33 +11,29 @@ The following plugins implement filtering features:
 
```

Comment:
"The filtering state management, Filter Row rendering, and filtering logic are implemented in the following plugins:" and then
"The following plugins implement filtering features:"? Why do you repeat this?

Reply:
Looks like there is no need in the last sentence of the first paragraph. I'll remove it. Thanks!